ha/k8s_setup/ha_start.py

```diff
@@ -26,44 +29,65 @@ def usage(prog: str):
         """
         Print usage instructions
         """
-        sys.stderr.write(
-            f"usage: {prog} [-h] <--config url> <--services service to start>...\n"
+        print(
+            f"usage: {prog} [-h] <-s/--services service to start> <-c/--config url> [-d/--debug [0|1]]...\n"
             f"where:\n"
-            f"--services health_monitor, fault_tolerance, k8s_monitor\n"
-            f"--config   Config URL\n")
+            f"-s --services health_monitor, fault_tolerance, k8s_monitor\n"
+            f"[-c --config   Config file URL : yaml://<file_path>] E.g. yaml:///etc/cortx/cluster.conf\n"
+            f"[-d --debug   [level]] (if level is not provided, then it considered as 1)"
+            f"\n\t\t0 : No debug.\n\t\t1 : Auto restarts child process if it dia.", file=sys.stderr, flush=True)
 
 def handle_signal(signum, frame):
     """
     Signal handler call beck function, to forward the received signal to driver process
     """
-    sys.stdout.write(f"Signal {signum} at frame {frame} received.")
-    sys.stdout.write(f"Sending {signum} to driver process pid: {driver_process.pid}.")
+    print(f"Signal {signum} at frame {frame} received.", flush=True)
```

Comment:
Any concrete reason to use flush all over?

Reply:
I restarted the health monitor 2 3 times see no log at all, so searched and used sys.stdout.flush() in between but it doesn't work.
so if there is overly less log then for a long time you will not able to see any log so this will not give us an error at the instance in any in the log of driver process. hence I need to add a flush option to all. the setting flush option is the only solution for me

Reply:
Ok..